### PR TITLE
ClinSeq SummarizeCnvs uses test data for its test.

### PR DESCRIPTION
**Pull request by tmooney**
Here's another test affected by the deletion of abandoned data.

### CODE REVIEW

**Comment by wschroeder** [comment]
+1 :radio_button:

**Comment by jasonwalker80** [comment]
+1

File: lib/perl/Genome/Model/ClinSeq/Command/SummarizeCnvs.t [hunk 1]
```diff
@@ -14,30 +14,32 @@ BEGIN {
...
 #Create a temp dir for results
 my $temp_dir = Genome::Sys->create_temp_directory();
 ok($temp_dir, "created temp directory: $temp_dir") or die;
 
 #Get a clin-seq build
-my $clinseq_build_id = 126680687;
+my $data = Genome::Model::ClinSeq::TestData->load();
+my $clinseq_build_id = $data->{CLINSEQ_BUILD};
 my $clinseq_build    = Genome::Model::Build->get($clinseq_build_id);
 ok($clinseq_build, "obtained a clinseq build from the database for build id: $clinseq_build_id") or die;
```
Comment (susannasiebert) [inline lib/perl/Genome/Model/ClinSeq/Command/SummarizeCnvs.t]: This test message is outdated since this is now mock data.

**Comment by susannasiebert** [comment]
+1

File: lib/perl/Genome/Model/ClinSeq/Command/SummarizeCnvs.t (hunk 1 above)
Comment (tmooney) [inline lib/perl/Genome/Model/ClinSeq/Command/SummarizeCnvs.t]: Indeed... updated!

**Pull request merged**